src/main/java/bio/terra/testrunner/runner/TestRunnerEnvironmentScript.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.testrunner.runner;
+
+import java.util.Map;
+
+public class TestRunnerEnvironmentScript {
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public TestRunnerEnvironmentScript() {}
+
+  /**
+   * Setter for any parameters required by the GitHub script. These parameters will be set by the
+   * Test Runner based on the current Server Configuration, and can be used by the GitHub script
+   * methods.
+   *
+   * @param parametersMap map of string key-value pairs supplied by the server.versionScripts
+   *     configuration
+   */
+  public void setParametersMap(Map<String, String> parametersMap) throws Exception {}
```

Comment:
It looks like this is unused, are you expecting other scripts to use it in the future?

Reply:
It could be used by a future custom run time script that requires parameters.